### DISCUSSION TITLE
Make filters persistent

### DISCUSSION THREAD

**Discussion by internetbug256** [laravel/nova-issues#4389] [Ideas & Feature Requests]
Hello. I see that lot of people, while Nova 3.x was the latest version, used the Detached filter (https://github.com/outl1ne/nova-detached-filters) in order to have their filters to persist between submissions.
Now that version 4.0 is out, and the above plugin is not compatible, and there are no plans from the author to update it, I am really lost.

So I am begging you to p-p-p-p-p-please consider to have a way to make filters persistent...
Thank you

PS: this has also been discussed in #3123, but for some reason not taken in count...

**Comment by vesper8** [comment]
Just want to mention that https://github.com/outl1ne/nova-detached-filters works on Nova 4.. it's one of my favourite packages and I use it extensively.
